sections/shared/modals/WalletOptionsModal/WalletOptionsModal.tsx

```diff
@@ -195,11 +205,32 @@ const WalletOptionsModal: FC<WalletOptionsProps> = ({ onDismiss, setWatchWalletM
 								onDismiss();
 								setWatchWalletModalOpened(true);
 							}}
+							data-testid="watch-wallet"
 						>
 							{searchIcon} {t('modals.wallet.watch-wallet.title')}
 						</StyledButton>
+						{isMainnet && (
+							<StyledButton
+								onClick={() => {
+									onDismiss();
+									setDelegateModalOpened(true);
+								}}
+							>
+								{delegateIcon} {t('modals.wallet.delegate-mode.menu-title')}
+							</StyledButton>
+						)}
 					</Buttons>
 					<StyledDivider />
+					{delegateWallet && (
```

Comment:
We should consider making the text have a defined width so the icons line up 

![image](https://user-images.githubusercontent.com/39738607/124866812-5542e000-e000-11eb-9134-717a84d9dc33.png)